wzdx/monitoring/experimental/main.py

```diff
@@ -18,7 +18,7 @@ def get_path():
     date = datetime.datetime.now(pytz.timezone("US/Mountain"))
     print(date)
```

Comment:
we've got logging imported here, but use quite a few "print" statements. Can we swap that over?

Reply:
Updated to logging statements!